### ISSUE TITLE
CSGO update

### ISSUE THREAD

**Issue by kirill-21** [danielkrupinski/Osiris#4235]
Steam updated GameOverlayRenderer.dll and osiris can not find it's patterns now, will it be updated for CSGO? A lot of people still play that game via csgo_legacy beta version)

**Comment by doubleOpink** [comment]
I agree with you, I really hope that Osiris for csgo will be updated

**Comment by ITSBONVOYAGE** [comment]
I also really hope that the author updates the cheat

**Comment by danielkrupinski** [comment]
Fixed in ef472aa
